standard/ERCs/erc-6900.md

```diff
@@ -471,72 +471,55 @@ An account can have the same validation module installed more than once.
 The entityId of a validation function installed on an account MUST be unique.
 Validation installation MAY be deferred until a later time, such as upon first use.
 
-#### Responsibilties of `StandardExecutor` and `ModuleExecutor`
-
-`StandardExecutor` functions are used for open-ended calls to external addresses.
-
-`ModuleExecutor` functions are specifically used by modules to request the account to execute with account's context. Explicit permissions are required for modules to use `ModuleExecutor`.
-
-The following behavior MUST be followed:
-
-- `StandardExecutor` can NOT call module execution functions and/or `ModuleExecutor`. This is guaranteed by checking whether the call's target implements the `IModule` interface via ERC-165 as required.
-- `StandardExecutor` can NOT be called by module execution functions and/or `ModuleExecutor`.
-- Module execution functions MUST NOT request access to `StandardExecutor`, they MAY request access to `ModuleExecutor`.
-
```

Comment:
this is not deleted or changed. Github just moved it to the end of the newly added sections.